test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -15,6 +15,7 @@ check_result_qemu() {
     umount $TESTDIR/root
     [[ -f $TESTDIR/failed ]] && cat $TESTDIR/failed
     ls -l $TESTDIR/journal/*/*.journal
+    journalctl -u testsuite --no-pager --file=$TESTDIR/journal/*/*.journal
```

Comment:
I'll take out the `journalctl` entry from this contribution, and I suggest to tackle this in a separate PR.